<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DungeonSubtileWidget</class>
 <widget class="QFrame">
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <!--property name="styleSheet">
   <string notr="true">QFrame { background-color: palette(window); }</string>
  </property>
  <property name="windowOpacity">
   <number>0.5</number>
  </property-->
  <property name="minimumWidth">
   <number>236</number>
  </property>
  <property name="maximumWidth">
   <number>236</number>
  </property>
  <layout class="QVBoxLayout">
   <!--item>
    <widget class="QWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout">
      <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
      <item alignment="Qt::AlignCenter">
       <layout class="QHBoxLayout" name="centerButtonsHorizontalLayout"/>
      </item>
      <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
      <item alignment="Qt::AlignRight">
       <layout class="QHBoxLayout" name="rightButtonsHorizontalLayout"/>
      </item>
     </layout>
    </widget>
   </item-->
   <item>
    <widget class="QWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1" colspan="2" alignment="Qt::AlignCenter">
       <layout class="QHBoxLayout" name="centerButtonsHorizontalLayout"/>
      </item>
      <item row="0" column="3" alignment="Qt::AlignRight">
       <layout class="QHBoxLayout" name="rightButtonsHorizontalLayout"/>
      </item>
      <item row="1" column="0" colspan="4" alignment="Qt::AlignCenter">
       <widget class="QLabel" name="monsterName"/>
      </item>
      <item row="2" column="0" colspan="4" alignment="Qt::AlignCenter">
       <widget class="QLabel" name="monsterLevel"/>
      </item>

      <item row="3" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Status:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="monsterStatus"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;AI-Int:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="monsterInt"/>
      </item>

      <item row="4" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Damage:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="monsterDamage"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;To Hit:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="monsterHit"/>
      </item>

      <item row="5" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Damage2:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="monsterDamage2"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;To Hit2:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="monsterHit2"/>
      </item>

      <item row="6" column="2">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Magic Hit:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="monsterMagic"/>
      </item>


      <item row="7" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Hit Points:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="monsterHp"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Armor:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="monsterArmorClass"/>
      </item>
      <item row="8" column="2">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Evasion:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLabel" name="monsterEvasion"/>
      </item>

      <item row="9" column="0">
       <widget class="QLabel">
        <property name="text">
         <!--string>&lt;b&gt;Phy. Res.:&lt;/b&gt;</string-->
         <string>&lt;b&gt;Resists:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="3">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QProgressBar" name="monsterResSlash">
          <property name="fixedWidth">
           <number>52</number>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="monsterResBlunt">
          <property name="fixedWidth">
           <number>52</number>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="monsterResPunct">
          <property name="fixedWidth">
           <number>52</number>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <!--item row="10" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Mag. Res:&lt;/b&gt;</string>
        </property>
       </widget>
      </item-->
      <item row="10" column="1" colspan="3">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QProgressBar" name="monsterResFire">
          <property name="fixedWidth">
           <number>38</number>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="monsterResLight">
          <property name="fixedWidth">
           <number>38</number>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="monsterResMagic">
          <property name="fixedWidth">
           <number>38</number>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="monsterResAcid">
          <property name="fixedWidth">
           <number>38</number>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="11" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Flags:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="3">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="monsterHiddenCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Hidden</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="monsterStealCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Steals life</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="monsterOpenCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Opens doors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="monsterSearchCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Searches for enemy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="monsterBleedCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Bleeds</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="monsterKnockbackCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Knocks back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="monsterNoStoneCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Immune to stone curse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="monsterGargCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Starts in a special mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="monsterNoDropCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>No loot</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>

      <item row="12" column="0" colspan="2">
       <widget class="QLabel">
        <property name="text">
         <string>&lt;b&gt;Xp Reward:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="monsterExp"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
